beta/examples/tensorflow/object_detection/models/yolo_v4_config.py

```diff
@@ -0,0 +1,26 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from beta.examples.tensorflow.common.sample_config import SampleConfig
+
+YOLOV4_CONFIG = SampleConfig({
+    "num_feature_layers": 3,
+    "num_classes": 80,
+    "model_params": {
+        "architecture": {
+            "backbone": {
+                "name": "darknet"

```

Comment:
Does Yolo v4 support another backbone? I mean we have resnet-50 backbone.  What happens if I set "resnet" as the backbone in the config.

Reply:
It is possible to enable support for other backbones in the future. For now, we support only the darknet backbone. Fix: the ValueError will raise in the case of the other backbone is set.